pydm/main_window.py

```diff
@@ -1,6 +1,7 @@
 import os
 from os import path
 
+from qtpy import QtCore
 from qtpy.QtWidgets import QApplication, QMainWindow, QFileDialog, QAction, QMessageBox
 from qtpy.QtCore import Qt, QTimer, Slot, QSize, QLibraryInfo
```

Comment:
we seems to be importing qtpy.QtCore here. could QCoreApplication be added to this?